src/components/placeCreate/search/PlacePreviewMap.tsx

```diff
@@ -16,13 +18,22 @@ const PlacePreviewMap = ({
         lng: longitude,
       }}
       isPanto={true}
-      draggable={false}
+      draggable={draggable}
       level={level}
       style={{
         width,
         height,
       }}>
-      <MapMarker position={{ lat: latitude, lng: longitude }} />
+      {mapMarkers ? (
+        mapMarkers.map((marker) => (
```

Comment:
p5;
```suggestion
        mapMarkers.map(({id, latitude, longitude}) => (
```
이렇게 쓰면 어떨까요?? 가독성이 좀 좋아질것같아요~
저도 지금까지 유리님처럼 썼는데 리팩터링하면서 하나씩 바꿔야겠어요.. 😅

Reply:
p1으로 바꾸겠습니다 바로 반영할게융

Reply:
https://github.com/prgrms-web-devcourse/Team-5YES-WuMo-FE/pull/51/commits/b442ab760d96a1d3b9b9b0b5f72295a8a1c7425c